docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -55,20 +55,18 @@ components:
 ----
 ====
 
-This example has two containers, each defining a single endpoint. The endpoint is an accessible port that can be made accessible inside the workspace. Each endpoint has a name and port, which is the port on which the server running inside the container is listening. See the following attributes that you can set on the endpoint:
+This example has two containers, each defining a single endpoint. The endpoint is an accessible port that you can make accessible inside the workspace. Each endpoint has a name and port. The server running inside the container is listening on this port. See the following attributes that you can set on the endpoint:
 
 * `exposure`: When its value is `public`, the endpoint is accessible outside the workspace and is exposed on port `80` or `443` depending on whether TLS is enabled in devfiles. Access this endpoint from the devfile user interface.
 
 * `protocol`: For public endpoints, the protocol is a hint to the UI on how to construct the URL for the endpoint access. Typical values are `http`, `https`, `ws`, `wss`.
```

Comment:
Devfiles don't have a UI, so is this `protocol` attribute still relevant for endpoints?

Reply:
it's still relevant https://github.com/devfile/api/blob/main/schemas/latest/devfile.json#L318

devfiles does not have UI, but projects may expose endpoints url, like Che. And protocol is used to get the full URL constructed

Reply:
@sleshchenko, can we use, for example, "a hint to the UI of the application that consumes the devfile on how to..."?

Reply:
@rkratky sounds good. Or maybe just `devfile consumer`, like ` a hint to the devfile consumer on how to...`